tests/vnc_playback.html

```diff
@@ -2,8 +2,22 @@
 <html>
     <head>
         <title>VNC Playback</title>
-        <script src="/vendor/browser-es-module-loader/dist/browser-es-module-loader.js"></script>
-        <script type="module" src="./playback.js"></script>
+        <!-- promise polyfills promises for IE11 -->
+        <script src="../vendor/promise.js"></script>
+        <!-- ES2015/ES6 modules polyfill -->
+        <script type="module">
+            window._noVNC_has_module_support = true;
+        </script>
+        <script>
+            window.addEventListener("load", function() {
+                if (window._noVNC_has_module_support) return;
+                var loader = document.createElement("script");
+                loader.src = "vendor/browser-es-module-loader/dist/browser-es-module-loader.js";
```

Comment:
I retract my previous statement. This line is wrong. An extra `../` is needed here.